src/libraries/System.Linq.Queryable/src/System.Linq.Queryable.csproj

```diff
@@ -1,6 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <EnableTrimAnalyzer>true</EnableTrimAnalyzer>
```

Comment:
Do we not have trim analyzer enabled globally?

Reply:
You're right -- this is probably unnecessary.